src/modules/mahjong/HandValidator/HandValidator.ts

```diff
@@ -20,6 +20,61 @@ class HandValidator {
 
   public static REQUIRED_NUM_OF_MELDS = 4;
 
+  /**
+   * Validates the Length of the hand must be 14
+   * @param tiles An array of tile objects
+   * @returns boolean, true if hand length is 14, otherwise false
+   */
+  public static validateHandLength(tiles: Tile[]): boolean {
+    if (tiles.length !== 14) return false;
+    return true;
+  }
```

Comment:
Are we still not factoring in seasons/flowers?

Reply:
I think I only used this function for validateThirteenTerminals. But we're going to have to somehow deal with seasons/flowers. These tiles are going to be 'played' tiles. I'm thinking when we get tiles from the frontend we're going to need some property for played tiles. Probably going to have to ignore flowers/seasons. Will need to refactor later, but after we figure out how to handle that logic.